src/clib/lib/job_queue/job_node.cpp

```diff
@@ -177,7 +135,7 @@ void job_queue_node_set_status(job_queue_node_type *node,
         return;
 
     logger->debug("Set {}({}) to {}", node->job_name, node->queue_index,
-                  job_status_names.at(new_status).c_str());
+                  job_status_names.at(new_status));
```

Comment:
debug takes std::string and not char * ?

Reply:
I think it works with either through pybind.

Reply:
Probably.